npm-packages.md

```diff
@@ -0,0 +1,46 @@
+# Publishing and using `@guardian` NPM packages
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Using
+
+-   Make sure you include any `@guardian` packages for transpilation in your build process
+    -   e.g. your [`webpack.config.js`](https://github.com/webpack/webpack/issues/2031#issuecomment-219040479) might including something that looks like this:
```

Comment:
Could we also add examples for tooling such as storybook and jest? Maybe we examples of the sorts of errors that can be expected to be seem if this isn't done?

I've often hit this issue in multiple reports for multiple tools and even though I've fixed it several times it's easy to forget to tell the config to not exclude guardian packages and be faced with confusing error messages. I can imagine other people could have similar problems.

I think the motivation and reasoning behind this appraoch is sound and agree that we should do it but it is, nonetheless, counter intuative. We're all so used to npm packages being compiled down to something we don't need to worry about; we have all been blithly ignoring `node_modules` for so long it's become a given, we have coding muscle memory for it.

Reply:
> Could we also add examples for tooling such as storybook and jest?

yeah great idea, you ok to raise a pr?

>we have all been blithly ignoring node_modules for so long it's become a given, we have coding muscle memory for it

this was brought up in infra meetings actually, and we agreed we should really be transpiling everything, beucase you have no control over what's published by other people. but it's not formalised here – it probably should be